### DISCUSSION TITLE
希望增加举牌的功能

### DISCUSSION THREAD

**Discussion by fakers777** [FloatTech/ZeroBot-Plugin#712] [Ideas]
功能名称：举牌
实现：有些wx机器人qq机器人有这个功能，输入一段文字，然后出现内置的一副图片，比如，一行小人，每个人举一个牌子，上面写着输入的文字。图片模板有很多种，美女，小人，还有各类的

谢谢

**Comment by ping1145** [comment]
如何开始使用
